hypervisor/dm/vpci/vmcs9900.c

```diff
@@ -167,3 +168,46 @@ const struct pci_vdev_ops vmcs9900_ops = {
 	.write_vdev_cfg = write_vmcs9900_cfg,
 	.read_vdev_cfg = read_vmcs9900_cfg,
 };
+
+
+int32_t create_vmcs9900_vdev(struct acrn_vm *vm, struct acrn_emul_dev *dev)
+{
+	uint32_t i;
+	struct acrn_vm_config *vm_config = get_vm_config(vm->vm_id);
+	struct acrn_vm_pci_dev_config *dev_config = NULL;
+	int32_t ret = -EINVAL;
+	uint16_t vuart_idx = *((uint16_t*)(dev->args));
+
+	for (i = 0U; i < vm_config->pci_dev_num; i++) {
+		dev_config = &vm_config->pci_devs[i];
+		if (dev_config->vuart_idx == vuart_idx) {
+			spinlock_obtain(&vm->vpci.lock);
```

Comment:
I thought we concluded we didn't need this lock. Not?
@ywan170

Reply:
Yes. This lock can be removed.

Reply:
Sorry， will fix that